src/localization/tableStrings.js

```diff
@@ -50,45 +50,51 @@ export const tableStrings = new LocalizedStrings({
     supply_quantity: 'SUPPLY THIS\nINVOICE',
     their_stock: 'THEIR\nSTOCK',
   },
-  gil: { // TODO add: batch, batch_expiry, our_stock, supply_quantity, supplier
```

Comment:
I think `batch` is missing

Reply:
Oh, that's probably a new one. I'll add as TODO. Thanks!